tests/dummy/app/templates/application.hbs

```diff
@@ -8,6 +8,18 @@
   <div class="acceptance-feature-off"></div>
 {{/if-feature}}
 
+{{#if-feature 'feature-from-config'}}
+  <div class="feature-from-config-on"></div>
+{{else}}
+  <div class="feature-from-config-off"></div>
+{{/if-feature}}
+
 {{#if-feature 'off-feature'}}
   <div class="off-feature-on"></div>
 {{/if-feature}}
+
+{{#if features.flags.acceptance-feature}}
```

Comment:
This is in the template, but I don't believe any tests are using it. I'd like to keep bound flags out of the scope of this pr.